src/hooks/useGetColoniesMembersCount.ts

```diff
@@ -71,6 +75,27 @@ export const useGetColoniesMembersCount = (
     },
   });
 
+  const { data: newColonyContributorResult } =
+    useOnCreateColonyContributorSubscription();
+
+  const newColonyContributor =
+    newColonyContributorResult?.onCreateColonyContributor?.contributorAddress;
+
+  useEffect(() => {
+    let timeout: NodeJS.Timeout;
+
+    if (newColonyContributor) {
+      // It looks hacky, but we need the timeout to ensure that opensearch has been updated before we refetch.
```

Comment:
I know this fixes such a small edge case that it doesn't really matter too much, but is to `2000` timeout reliable for opensearch having updated? Does it reliably take less than 2 seconds?